### DISCUSSION TITLE
[Solved] How to show the correct time with timezone?

### DISCUSSION THREAD

**Discussion by rain2307** [JanDeDobbeleer/oh-my-posh#3548] [General]
I'm using WSL ubuntu on windows. Here is my config:
```
{ "style": "plain",
   "template": " <#b3b340>{{ dateInZone \"15:04:05\" .CurrentDate \"UTC+8\"}} ",
  "type": "time"
 },
```

I'm in china (UTC+8). The time is not correct. 
If I run `date` the result is "Thu Mar  2 22:32:40 CST 2023" which is not correct. But when I run ` sudo hwclock --show`, I got the correct time "2023-03-02 23:24:17.895815+08:00". 
I don't know if  it's the posh configuration or ubuntu setting or both that cause this issue？
Please help.

**Comment by JanDeDobbeleer** [comment]
This comes straight from golang's time library so if it wrong that one reports the incorrect time (not sure why).

**Reply by rain2307** [reply]
> This comes straight from golang's time library so if it wrong that one reports the incorrect time (not sure why).

Thanks for your comment. I just find that time is correct on my windows posh. So I think it's the same reason why `date` result is incorrect too. My english is poor...

**Comment by rain2307** [comment]
This  is an issue with wsl on windows. Time will be synced after reboot.
